### PR TITLE
docs: added _Viewing and selecting the current Kubernetes context_

### PR DESCRIPTION
**Pull request by themr0c**
Added a procedure explaing how to set the current Kubernets context.

Screenshot from 2023-05-10 13-02-13
Screenshot from 2023-05-10 13-02-03
Screenshot from 2023-05-10 13-01-52

refs: https://github.com/containers/podman-desktop/issues/1449

### CODE REVIEW

**Review by jeffmaury — COMMENTED** [review]
Should'nt we have a Windows version of the images and also document how to access the Podman Desktop tray ?

File: website/docs/kubernetes/viewing-and-selecting-current-kubernete-context.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+tags: [migrating-to-kubernetes]
+---
+
+# Viewing and selecting the current Kubernetes context
+
+With Podman Desktop, you can view and select your current Kubernetes context.
+
+#### Prerequisites
+
+- You registered the Kubernetes cluster in your kubeconfig file: _<your_kubernetes_cluster>_. For example, [Creating a kind cluster](kind/creating-a-kind-cluster).
```
Comment (cdrage) [inline website/docs/kubernetes/viewing-and-selecting-current-kubernete-context.md]: This is a bit confusing? Couldn't we say something like: "You have a Kubernetes context configured within `~/.kube/config` ?

**Review by cdrage — DISMISSED** [review]

**Comment by themr0c** [comment]
I wish we would continue beautifying the screenshots in a next iteration.

**Review by jeffmaury — COMMENTED** [review]
What if the user has no current Kubernetes context. We should add this use case from my POV

**Comment by themr0c** [comment]
> What if the user has no current Kubernetes context. We should add this use case from my POV

See the prerequisites.

**Comment by jeffmaury** [comment]
> > What if the user has no current Kubernetes context. We should add this use case from my POV
> 
> See the prerequisites.

The prequisites are also my concern

**Comment by themr0c** [comment]
> The prequisites are also my concern

Do you mean: we need another procedure: registering a Kubernetes cluster?

**Review by jeffmaury — APPROVED** [review]
After reflection and re reading the title, this use case should be part of another doc

**Pull request merged**